### ISSUE TITLE
Logging in through google doesn't work?

### ISSUE THREAD

**Issue by NylaTheWolf** [gilgold/tabox#19]
I've tried logging in several times via Google, but nothing happens when I hit log in. I've tried several times on a few different days and it didn't work properly.

sorry for the brief post, I'm in class and gotta get to work haha

**Comment by gilgold** [comment]
Hey @NylaTheWolf can you tell me what version of Tabox you are using now and what browser? 
Thanks!

**Comment by NylaTheWolf** [comment]
> Hey @NylaTheWolf can you tell me what version of Tabox you are using now and what browser? Thanks!

Version 3.5 of Tabox and Google Chrome Version 94.0.4606.61 (Official Build) (64-bit)

**Comment by gilgold** [comment]
I made some improvements to the login process but I was never able to reproduce your issue, it always worked for me.
Once v3.5.1 is out please check and see if the issue is resolved, if not I will continue to investigate this.

**Comment by NylaTheWolf** [comment]
> I made some improvements to the login process but I was never able to reproduce your issue, it always worked for me. Once v3.5.1 is out please check and see if the issue is resolved, if not I will continue to investigate this.

Thank you for your efforts!

I think I found the possible issue: I turned off extension syncing a month or two ago, and I never turned it back on. 😅 I'll let you know if it works once I do.

**Comment by gilgold** [comment]
Hmm I wonder if that has something to do with it. Though Tabox itself does not relay on extension syncing to work, it's possible the Google auth does.

**Comment by NylaTheWolf** [comment]
So I finally got around to doing it, and after turning on syncing for my extensions, signing in through Google finally worked! However, I signed into Tabox on my Mac first, and it seems like the sessions on the Mac overrode the sessions I had on my Windows laptop because I don't see the sessions I saved on my Windows laptop. This isn't a huge deal at all for my case because I made sure to export every single window I had saved in Tabox on both Mac and Windows, so nothing was lost. I thought it'd be important to bring up though.

Thank you so much for your help!
